perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -164,27 +164,27 @@ public static void notNull(Object e, String name) throws InternalErrorException
 	}
 
 	/**
-	 * Define min length of some entity name.
+	 * Define min length of some property.
 	 *
-	 * @param name name of entity
+	 * @param propertyName name of checked property
 	 * @param minLength
 	 * @throws MinSizeExceededException
 	 */
-	public static void checkMinLength(String name, int minLength) throws MinSizeExceededException{
+	public static void checkMinLength(String propertyName, String actualValue, int minLength) throws MinSizeExceededException{
 
-		if(name.length()<minLength) throw new MinSizeExceededException("Length of name is too short! MinLength=" + minLength + ", ActualLength=" + name.length());
+		if(actualValue.length() < minLength) throw new MinSizeExceededException("Length of '" + propertyName + "' is too short! MinLength=" + minLength + ", ActualLength=" + actualValue.length());
 	}
 
 	/**
-	 * Define max length of some entity name.
+	 * Define max length of some property.
 	 *
-	 * @param name name of entity
+	 * @param propertyName name of checked property
 	 * @param maxLength
 	 * @throws
 	 */
-	public static void checkMaxLength(String name, int maxLength) throws MaxSizeExceededException{
+	public static void checkMaxLength(String propertyName, String actualValue, int maxLength) throws MaxSizeExceededException{
 
-		if(name.length()<maxLength) throw new MaxSizeExceededException("Length of name is too long! MaxLength=" + maxLength + ", ActualLength=" + name.length());
+		if(actualValue.length() > maxLength) throw new MaxSizeExceededException("Length of '" + propertyName + "' is too long! MaxLength=" + maxLength + ", ActualLength=" + actualValue.length());
```

Comment:
You should cover also the case, when passed value is ``null``, since ``Candidate`` params titleBefore/titleAfter can be null or empty.